server/api_link.go

```diff
@@ -444,14 +444,14 @@ func (s *ApiServer) LinkGoogle(ctx context.Context, in *api.AccountGoogle) (*emp
 
 	res, err := s.db.ExecContext(ctx, `
 UPDATE users
-SET google_id = $2, update_time = now()
+SET google_id = $2, display_name = $3, avatar_url = $4, update_time = now()
 WHERE (id = $1)
 AND (NOT EXISTS
     (SELECT id
      FROM users
      WHERE google_id = $2 AND NOT id = $1))`,
 		userID,
-		googleProfile.Sub)
+		googleProfile.Sub, googleProfile.Name, googleProfile.Picture)
```

Comment:
What's the format of Name and Picture? Does it work within the constraints of the columns in the database?

https://github.com/heroiclabs/nakama/blob/master/migrate/sql/20180103142001_initial_schema.sql#L23

Reply:
Nice observation. Both the fields are string, however length of the text could be a corner case that needs to be handled. Have revised the code to the following, which you will see in the next commit.

Created and used the following temp variables instead of the values received in googleProfile.
```
	displayName := googleProfile.Name
	if len(displayName) > 255 {
		// Trim the name in case it is longer than db can store
		displayName = displayName[0:255]		
	}
	
	avatarUrl := googleProfile.Picture
	if len(avatarUrl) > 512 {
		// Ignore the url in case it is longer than db can store
		avatarUrl = ""
	}
```